htdocs/compta/stats/cabyprodserv.php

```diff
@@ -180,7 +184,7 @@
 
 if ($modecompta == 'CREANCES-DETTES')
 {
-	$sql = "SELECT DISTINCT p.rowid as rowid, p.ref as ref, p.label as label,";
+	$sql = "SELECT DISTINCT p.rowid as rowid, p.ref as ref, p.label as label, p.fk_product_type as product_type,";
```

Comment:
If you introduce a new field into select with a group by, you must also add the field into the group by fields. Otherwise you get a syntax error with sql server that are strictly sql compliant.